pkg/network/ebpf/c/tracer/events.h

```diff
@@ -80,7 +80,7 @@ static __always_inline void cleanup_conn(void *ctx, conn_tuple_t *tup, struct so
         determine_connection_direction(&conn.tup, &conn.conn_stats);
     }
 
-    conn.conn_stats.timestamp = bpf_ktime_get_ns();
+    conn.conn_stats.duration = bpf_ktime_get_ns() - conn.conn_stats.duration;
```

Comment:
I find this line really confusing -- it might help to explain how `duration` is intended to be used. 

Subtracting a duration from a timestamp gets you a timestamp, but the field name is duration.

Reply:
Yes, the `duration` field is initialized to a timestamp so that we can calculate the duration of the connection here. I will add some comments.

Reply:
Yes that is the reason. I'll add a note.